Assets/Scripts/GameManagers/CameraManager.cs

```diff
@@ -0,0 +1,237 @@
+                }
+                break;
+        }
+    }
+
+    private static void UpdateCameraOrthographicSize()
+    {
+        currentLevel = getCurrentLevel();
+        float delta = Input.mouseScrollDelta.y;
+        float newHeight = mainCamera.orthographicSize + (delta * 0.1f);
+        float newWidth = newHeight * mainCamera.aspect;
+
+        float cameraX = mainCamera.transform.position.x;
+        float cameraY = mainCamera.transform.position.y;
+        float xAdjustment = 0, yAdjustment = 0;
+
+        // Prevent zoom from exceeding level size or zoom in limit
+        if (newHeight * 2.0f < GridManager.LEVEL_HEIGHT - 1 &&
+            newWidth * 2.0f < GridManager.LEVEL_WIDTH - 1 &&
+            newHeight > ZOOM_IN_LIMIT)
+        {
+            // Set new camera position if zooming out at edge of level
+            if (cameraX + newWidth > GridManager.mapLevels[currentLevel].getXMax())
+            {
+                xAdjustment = (cameraX + newWidth) - GridManager.mapLevels[currentLevel].getXMax();
+            }
+            else if (cameraX - newWidth < GridManager.mapLevels[currentLevel].getXMin())
+            {
+                xAdjustment = (cameraX - newWidth) - GridManager.mapLevels[currentLevel].getXMin();
+            }
+
+            if (cameraY + newHeight > GridManager.mapLevels[currentLevel].getYMax())
+            {
+                yAdjustment = (cameraY + newHeight) - GridManager.mapLevels[currentLevel].getYMax();
+            }
+            else if (cameraY - newHeight < GridManager.mapLevels[currentLevel].getYMin())
+            {
+                yAdjustment = (cameraY - newHeight) - GridManager.mapLevels[currentLevel].getYMin();
+            }
+
+            mainCamera.orthographicSize = newHeight;
+            mainCamera.transform.position = new Vector3(cameraX - xAdjustment, cameraY - yAdjustment, CAMERA_Z_POSITION);
+        }
+    }
+
+    public static void UpdateCamera()
+    {
+        if (Input.anyKey)
+        {
+            if (Input.GetKey(KeyCode.W)) // Move camera up
+            {
+                MoveCamera(CameraMovement.UP);
+            }
+
+            if (Input.GetKey(KeyCode.D)) // Move camera right
+            {
+                MoveCamera(CameraMovement.RIGHT);
+            }
+
+            if (Input.GetKey(KeyCode.S)) // Move camera down
+            {
+                MoveCamera(CameraMovement.DOWN);
+            }
+
+            if (Input.GetKey(KeyCode.A)) // Move camera left
+            {
+                MoveCamera(CameraMovement.LEFT);
+            }
+
+            if (Input.GetKeyDown(KeyCode.Q)) // Move camera up 1 level
+            {
+                MoveCamera(CameraMovement.LEVEL_UP);
+            }
+
+            if (Input.GetKeyDown(KeyCode.E)) // Move camera down 1 level
+            {
+                MoveCamera(CameraMovement.LEVEL_DOWN);
+            }
+        }
+
+        if (Input.mouseScrollDelta.y != 0) // Camera Zoom
```

Comment:
If we get to implementing a settings page with user preferences and custom key bindings, we should implement an option to invert the mouse wheel control of the camera so zoom in = scroll up and zoom out = scroll down.